packages/style-engine/class-wp-style-engine-css-declarations.php

```diff
@@ -34,9 +34,6 @@ class WP_Style_Engine_CSS_Declarations {
 	 * @param array $declarations An array of declarations (property => value pairs).
 	 */
 	public function __construct( $declarations = array() ) {
-		if ( empty( $declarations ) ) {
-			return;
-		}
```

Comment:
If empty, then `add_declarations` won't add anything. This check doesn't serve any purpose.